hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -47,6 +51,10 @@ public static HawkBuilder init(Context context) {
       return remove(key);
     }
 
+    if (!isHawkInitialised()) {
+      return false;
```

Comment:
a boolean would be enough, since either system is initialized or not. And in order to prevent logical mistake, exception would be good here. Init must be mandatory and it should be error that system shouldn't continue. Otherwise clients could interpret it wrong and make mistakes such as putting (if (not null) do things). You can also move init part to the top of the method, since it's the most important one. And since we use it in a few places, would be good to move it to a helper method and use it.